ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -31,6 +37,7 @@ final class CommandDeduplicationIT(timeoutScaleFactor: Double, ledgerTimeInterva
     "CDSimpleDeduplicationBasic",
     "Deduplicate commands within the deduplication time window",
     allocate(SingleParty),
+    runConcurrently = false,
```

Comment:
Why is this needed, to serialize on the time model update? How much slower does this make the build?

Reply:
Yes, because we change the time model.
Not sure how much time it will take but we already have other tests doing this and I don't really see any other way around it.